components/eventing-controller/controllers/backend/reconciler_test.go

```diff
@@ -667,7 +667,7 @@ func bebSecretExists(ctx context.Context) bool {
 	return len(secretList.Items) > 0
 }
 
-// creates a secret containing the oauth2 credentials that is expected to be
+// createOAuth2Secret creates a secret containing the oauth2 credentials that is expected to be
```

Comment:
(nit)
in case you want to improve the comment, you could also add point to the end of the comment as per go guidelines.